extensions/amp-list/0.1/amp-list.js

```diff
@@ -519,6 +517,16 @@ export class AmpList extends AMP.BaseElement {
     switch (previousLayout) {
       case Layout.RESPONSIVE:
         this.element.classList.remove('i-amphtml-layout-responsive');
+        setStyles(this.element, {
+          height: '',
+          width: '',
+        });
+        break;
+      case Layout.FLEX_ITEM:
+        setStyles(this.element, {
+          height: '',
+          width: '',
+        });
```

Comment:
Actually I realized that with the new CSS change, I can alter the overflowElement change below too as it is no longer necessary. It no longer works too, since `toggle` now uses visibility via the `hidden` attribute instead of `display: none`.

Reply:
Oh nvm. The CSS doesn't apply to the `overflow` attribute. Reverted that particular change after testing.